### ISSUE TITLE
203 Error with Custom Connector 

### ISSUE THREAD

**Issue by reneeloalm** [microsoft/coe-starter-kit#3124]
### Describe the issue

In multiple environments, the custom connector with verified client id, secret, tenant id, etc keeps throwing a 203 error when test connection. 
The app then throws errors throughout like GetBuilds failed.  

### Expected Behavior

Custom Connector tests successfully and no errors in Canvas app

### What component are you experiencing the issue with?

ALM Accelerator Custom Conenctor

### What solution version are you using?

1.0.20220517.1

### Steps To Reproduce

Test custom connector that is installed

**Comment by mikefactorial** [comment]
@reneeloalm Possibly related to #3100? Can you take a look at the troubleshooting we've done on that issue and report back. Different HTTP status code but could be related.

**Comment by reneeloalm** [comment]
I updated settings based on other issue.  One note as well both environments I have issue in are GCC in case that matters. 

 I just tested again with same 203 result.  I used the swagger editor to test GetOrganizations call with no parameters and I get this error.  <html>
<body>


Failed to fetch.Possible Reasons:CORSNetwork FailureURL scheme must be "http" or "https" for CORS request.
--



</body>
</html>
